sdk/inc/azure/core/az_mqtt5_property_bag.h

```diff
@@ -0,0 +1,388 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ *
+ * @brief This header defines the MQTT 5 property bag and its related methods. The following
+ * structures and API are used by the SDK when sending and receiving MQTT5 properties.
+ *
+ * @note A property bag struct contains an internal property bag options struct. These options
```

Comment:
nit: not internal anymore/not contained in the property bag(?)

Reply:
Removed, thanks for pointing that out!